wrappers/hnynethttp/nethttp.go

```diff
@@ -156,7 +173,16 @@ func (ht *hnyTripper) spanRoundTrip(ctx context.Context, span *trace.Span, r *ht
 	}
 	span.AddField("meta.type", "http_client")
 	span.AddField("name", "http_client")
-	r.Header.Add(propagation.TracePropagationHTTPHeader, span.SerializeHeaders())
+	if ht.propagationHook == nil {
```

Comment:
comment here would be great
```suggestion
	if ht.propagationHook == nil {
		// if no propagation is set, default to the Honeycomb header format
```